icu4c/source/samples/legacy/README

```diff
@@ -1,44 +0,0 @@
-Copyright (C) 2016 and later: Unicode, Inc. and others.
-License & terms of use: http://www.unicode.org/copyright.html
-
-Copyright (c) 2002, International Business Machines Corporation and others. All Rights Reserved.
-This example demonstrates running an instance of  ICU 1.8.1. together with a current version of ICU. It only tests u_getVersion and several collation APIs. 
-
-Generally, one should be able to simultaneously use one or more versions of ICU 2.0 or higher and one version of ICU 1.8.1 or lower. 
-
-What is it all about: Let's say you have a 10 Tb database indexed using ICU 1.8.1. sortkeys. New ICU comes out, with neat new features you would like to use, but also with new sortkeys and you don't care to reindex your 10 Tb database. What to do then??? You can use ICU 1.8.1. in one of your compilation units and current version in all the others. So, you can use old collation until you decide to reindex. 
```

Comment:
This is actually useful because we, and some of our users, still rely on this functionality. It's why we have urename.h.

This kind of sample does require two versions of ICU to build against. One of them need not be 1.8.1.

Reply:
Let's please keep this one for now.

Reply:
should be easy enough to get this behavior using actions.  In fact, the 'old' version of ICU could come from `libicu-dev` or such from a package manager if building on ubuntu-latest.